rosdep/python.yaml

```diff
@@ -149,6 +149,19 @@ exhale-pip:
   ubuntu:
     pip:
       packages: [exhale]
+geomag:
```

Comment:
Mind using the key name `geomag-pip`?

https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1

Reply:
Also I see the contributing guide asks for `python-` and `python3-` suffixes. I guess that means this should be `python-geomag-pip` and `python3-geomag-pip`.

 @ros/rosdeputies Why is that the rule? Whether it's Python 2 or Python 3 depends on the version of python set in `ROS_PYTHON_VERSION` or the default rosdistro, what's the value of separate `python-` and `python3-` prefixes for keys only on PyPI?

Reply:
c1858da is where I add `-pip` suffix. Let me know what you want for a prefix and I'll add it. I have the same questions you do about the prefix.

Reply:
Opening PR to change recommendation: https://github.com/ros/rosdistro/pull/25508

Reply:
I'm sorry for the back and forth on this, but in #25508 we eventually landed on putting the `python3-` prefix on the front and the `-pip` suffix on the back.  @tdenewiler Would you mind updating one more time to conform to that?  Then I think we're good to go here.

Reply:
That was a good discussion in #25508, and this approach makes sense to me. Package management is not easy, I appreciate the thought going into this. I made the requested updates.

Reply:
> That was a good discussion in #25508, and this approach makes sense to me. Package management is not easy, I appreciate the thought going into this. I made the requested updates.

Perfect, thanks for your patience.